json_serializable/build.yaml

```diff
@@ -0,0 +1,8 @@
+builders:
```

Comment:
We should add a `# some link here` comment to point source diggers to the right spot to understand how this is suppose to work?

Reply:
Done